### ISSUE TITLE
A Hint on Veg layers from RCAT?

### ISSUE THREAD

**Issue by joewheaton** [Riverscapes/sqlBRAT#28]
So I was in over in [RCAT seeing how it did in RAVE](https://github.com/Riverscapes/RCAT/issues/24), and discovered that for one of the old ETAL RCAT RS Projects, the existing cover was coming in with symbology preserved through RAVE.  May be worth exploring why @philipbaileynar.

![image](https://user-images.githubusercontent.com/9723905/81106524-061c8f00-8ed3-11ea-807a-d1733bde4aca.png)

Video:
[![](http://img.youtube.com/vi/aPa3qBNZHnY/0.jpg)](http://www.youtube.com/watch?v=aPa3qBNZHnY "")

**Comment by philipbaileynar** [comment]
I don't think so. I checked and RAVE definitely has no symbology for veg. Unless you accidentally have some in %AppData%

What I think is going on here is raster [side car files](https://en.wikipedia.org/wiki/Sidecar_file). I suspect there's a lookup table DBF or other file beside the raster together with some color palette information that tells ArcGIS to use that info as the default symbology. 

We should look into this technique because it could be useful for us either for tools to write these files, or RAVE to store and read symbology this way.

**Comment by joewheaton** [comment]
Okay... I will take a closer look.
